### PR TITLE
Issue #13345: Enable MultipleVariableDeclarationsCheck example tests

### PR DESCRIPTION
**Pull request by absolutely-sam**
Issue : #13345: Enable MultipleVariableDeclarationsCheck example tests

### CODE REVIEW

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+*/
+package com.puppycrawl.tools.checkstyle.checks.coding.multiplevariabledeclarations;
+
+// xdoc section -- start
+public class Example1 {
+  public void myTest() {
+    int mid = 0;
+    int high = 0;
+
+    int lower, higher; // violation, multiple variables declared in same statement
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java:16]: This Check has two violations https://checkstyle.org/checks/coding/multiplevariabledeclarations.html#Violation_Messages
It is not clear what violation is there, please use part of violation message 
`// violation, 'variable declaration must be in its own statement'`
Or ir something similar

**Review by romani — CHANGES_REQUESTED** [review]
Direction is good, thanks a lot.

Please fix CI.

Items:

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java (hunk 1 above)
Comment (absolutely-sam) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java:16]: Would the following change address the issue? Please let me know if this works or if any further adjustments are needed.

```
int lower, higher;// violation, variables declaration lower, higher
    // must be in its own statement
```

Comment (romani) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java]: You missed single quotes

**Comment by romani** [comment]
Before pushing to GitHub, make sure `mvn clean verify ` pass on your local, if not passing, please investigate and fix that it points to

**Comment by absolutely-sam** [comment]
@romani 
Thanks for the help, `mvn clean verify` command is successful in my local.

I am not able to figure out the reason for the pipeline failure for the error in the image. Could you help me understand this error

image

**Comment by romani** [comment]
Please apply change that you have on your local, it should be part of commit.

**Comment by romani** [comment]
single commit always, please.

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java (hunk 1 above)
Comment (absolutely-sam) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/coding/multiplevariabledeclarations/Example1.java]: Addressed

**Comment by romani** [comment]
Please squash all commits in one

**Review by romani — APPROVED** [review]
Ok to merge if CI pass

**Comment by romani** [comment]
@absolutely-sam , please do one more fix for another Check.

Thanks a lot in advance!!

**Comment by absolutely-sam** [comment]
> @absolutely-sam , please do one more fix for another Check.
> 
> Thanks a lot in advance!!

Sure I will pick another Check and thank you for helping me.

**Pull request merged**